src/server/mod.rs

```diff
@@ -416,7 +413,7 @@ impl LanguageServer for LspServer {
                     true,
                 )),
                 execute_command_provider: Some(lsp::ExecuteCommandOptions {
-                    commands: vec![LspServerCommand::InjectTagFilter.into(), LspServerCommand::InjectTagValueFilter.into(), LspServerCommand::InjectFieldFilter.into(), LspServerCommand::InjectMeasurementFilter.into()],
+                    commands: vec![],
                     work_done_progress_options: lsp::WorkDoneProgressOptions {
                         work_done_progress: None,
                     }
```

Comment:
Before @rockstar added the inject* commands, the `execute_command_provider` was set to `None`. I'm not sure whether I should change it back to `None` or just leave an empty array here.

Reply:
This is where the LSP tells the monaco-editor, on the jsonrpc `initialize` request, what executeCommands it supports. Since we directly message the LSP from the UI (instead of going through monaco-editor), we are not currently telling the monaco-editor about the schema composition commands.

 @rockstar is trying to make the monaco-editor the src of all event coordination. The question is whether or not he plans to try making the executeCommands be triggered through the monaco-editor as well. If he is, then we should be adding the schema composition commands here.

If he isn't...then we could make this `None` again.